appinventor/components/src/com/google/appinventor/components/runtime/PasswordTextBox.java

```diff
@@ -54,4 +58,19 @@ public PasswordTextBox(ComponentContainer container) {
     view.setImeOptions(EditorInfo.IME_ACTION_DONE);
 
   }
+  
+  @SimpleProperty
```

Comment:
@ColinTree Please set a description here so that there is a tooltip on the block.